test/Microsoft.AspNetCore.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.OrderUsingHtmlHelpers.Encoded.html

```diff
@@ -1,11 +1,10 @@
-﻿<html>
+<html>
 <head>
     <meta charset="utf-8" />
     <title></title>
 </head>
 <body>
-<form action="HtmlEncode[[/UrlEncode[[HtmlGeneration_Order]]/UrlEncode[[Submit]]]]" method="HtmlEncode[[post]]">
-        <div>
+<form action="HtmlEncode[[/UrlEncode[[HtmlGeneration_Order]]/UrlEncode[[Submit]]]]" method="HtmlEncode[[post]]">        <div>
```

Comment:
Actually, there is no tag helpers involved here. It is because of the `<text>` tag here.
```HTML
@* Use comments and <text/> elements to force whitespace in generated HTML. *@@using (
Html.BeginForm(actionName: "Submit", controllerName: "HtmlGeneration_Order"))
    {<text></text>
        <div>
```